README.md

```diff
@@ -53,8 +60,16 @@ python3 build_tfm.py -m ARM_MUSCA_B1 -t GNUARM -c ConfigPsaApiTestIPC.cmake -s C
 Note: Make sure the TF-M Regression Test suite has PASSED on the board before
 running any PSA Compliance Test suite to avoid unpredictable behavior.
 
+To display help on supported options and targets:
+```
+python3 build_psa_compliance.py -h
+```
+
 ## Building the Mbed OS application
 
+After building the TF-M regression or PSA compliance tests for the target, it should be
+followed by building a Mbed OS application.
```

Comment:
The text could be more precise. We are now building the main application of this repo which will execute the test suites previously built.
The next step missing is how to execute the tests i.e. flash the binary to the target.

Reply:
done

Reply:
That's very detailed, thanks!
I was asking for the next step because the following section is `test_psa_target.py` which is yet another thing.